lib/Runtime/Library/InJavascript/Array_prototype.js

```diff
@@ -92,31 +86,31 @@
         return new __chakraLibrary.ArrayIterator(arrayObj, iterationKind);
     });
 
-    platform.registerFunction(platform.FunctionKind.Array_keys, function () {
+    platform.registerFunction('keys', function () {
```

Comment:
I removed the `FunctionKind` object instead supplying the name of each function - as the target (Array.prototype) is now detected based on the file name so didn't need to convey extra information.